tests/test-components/PastCommitNode.spec.tsx

```diff
@@ -29,13 +31,15 @@ describe("PastCommitNode", () => {
       is_remote_branch: false,
       name: "name3",
       upstream: "upstream",
+      top_commit: "2414721b194453f058079d897d13c4e377f92dc6",
       tag: "v1.0.4-14-g2414721"
     },
     {
       is_current_branch: false,
       is_remote_branch: true,
       name: "name4",
       upstream: "upstream",
+      top_commit: "2414721b194453f058079d897d13c4e377f92dc6",
```

Comment:
WDYT about adding some assertions on this new `prop` so that its intended side-effect is clear?

Reply:
Talked offline with @jaipreet-s , all the test cases are already using the new prop, so there is no need to add any more assertions.